overlord/snapstate/backend/link.go

```diff
@@ -153,10 +149,28 @@ func updateCurrentSymlinks(info *snap.Info) (revert func(), e error) {
 	return revertFunc, nil
 }
 
+// MaybePrepareReboot configures the system for a reboot if necesssary because
+// of a snap refresh. isUndo must be set when we are installing the previous
+// snap while performing a revert of the latest one that was installed
+func (b Backend) MaybePrepareReboot(info *snap.Info, dev snap.Device, isUndo bool) (boot.RebootInfo, error) {
```

Comment:
my preference is to follow boot and calls this MaybeSetNextBoot

Reply:
Changed